charts/k8s-monitoring/destinations/otlp-values.yaml

```
@@ -229,6 +229,24 @@ processors:
       resource: []
       span: []
       spanevent: []
+
+  filter:
+    enabled: false
+    # -- Metric filters
+    # @section -- Filter Processor
+    metrics:
+      metric: []
+      datapoint: []
+    # -- Log filters
+    # @section -- Filter Processor
+    logs:
+      logRecord: []
```

```suggestion
      log_record: []
```

we should match what we have in `feature-application-observability/values.yaml`

---

Yeah... but I want to standardize somewhere and IMO we've got way more camel case than snake case